src/app/story/story.component.ts

```diff
@@ -98,6 +98,9 @@ export class StoryComponent implements OnInit {
     } else if (this.story) {
       this.story.loadRelated('distributions').subscribe(() => {
         this.distPodcast = this.story.distributions.some(d => d.kind === 'episode');
+        if (this.distPodcast) {
+          this.story.distributions.find(d => d.kind === 'episode').loadRelated('episode');
```

Comment:
I found that if I did not loadRelated on the episode up front, the status box would not show that the episode URL field is required until navigating to the Podcast Distribution tab.